package/toltec-bootstrap/toltecctl

```diff
@@ -465,7 +465,7 @@ reinstall-root() {
         if [[ -v "on_root_packages[$pkgname]" ]]; then
             reinstall_packages[$pkgname]=1
         fi
-    done < <(gunzip -c $toltec_dest/var/opkg-lists/* | grep "^Package:" | awk '{print $2}')
+    done < <(gunzip -c "$toltec_dest"/var/opkg-lists/* | grep "^Package:" | awk '{print $2}')
```

Comment:
And some cases below ;-)

Reply:
Line 693, and possibly others. Too tired to catch all now.

Reply:
Just did a full pass, they should all be covered now.